android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -13,6 +13,14 @@ import java.util.concurrent.TimeUnit.SECONDS
 
 val androidHome: String by lazy { System.getenv("ANDROID_HOME") }
 val adb: String by lazy { "$androidHome/platform-tools/adb" }
+private val buildTools: String? by lazy {
+    File(androidHome, "build-tools")
+            .listFiles()
+            .sortedArray()
+            .lastOrNull()
+            ?.absolutePath
+}
+val aapt: String by lazy { buildTools?.let { "$buildTools/aapt" } ?: "" }
```

Comment:
do we want to allow empty strings?

Now I'm looking at this api and can't really say that it was  a good decision to expose such paths through `val`s…

Maybe we should use Rx here? Like `val aaptPath: Maybe<String>`?

Reply:
Well, was thinking about that, too. 
Not sure what the good way here would be, cause anyway users will need to process wrong/empty/undefined paths. The same applies to `adb` and `androidHome`...

Reply:
Im gonna merge this, since its needed for composer-related issues.
But lets continue discussing the best option - Maybe<>, Koptional, Nullable or empty string for the errors here.